stack.py

```diff
@@ -35,9 +37,28 @@ def eval_program(program):
             elif opcode == "divide":
                 c = b / a
             stack.append(c)
+        elif opcode == "swap":
+            # "swap" aliases to "swap 1"
+            swap_gap = get_argument(instuction, 1)
+            stack[-1], stack[-(1 + swap_gap)
```

Comment:
Please don't split lines like this in the future.
This is probably clearer as something like
```python
from, to = -1, -(1 + swap_gap)
stack[from], stack[to] = stack[to], stack[from]
```